source/adios2/toolkit/remote/EVPathRemote.cpp

```diff
@@ -171,7 +171,6 @@ EVPathRemote::GetHandle EVPathRemote::Get(char *VarName, size_t Step, size_t Blo
     GetMsg.Start = Start.data();
     GetMsg.Dest = dest;
     CMwrite(m_conn, ev_state.GetRequestFormat, &GetMsg);
-    CMCondition_wait(ev_state.cm, GetMsg.GetResponseCondition);
```

Comment:
Was this wait here to block until the write is done?

Reply:
Yes, but we were inconsistent.  The Get in EVPathRemote returned a GetHandle, which we could have Waited on, but instead we waited in the Get and never used the returned GetHandle.  This didn't matter much because there was no concurrency on the server side.  But there is possible concurrency on the Xrootd server side, so I've changed BP5Reader to actually send all the Gets and then wait on all the handles.  But I forgot to change the EVPath remote to match that.  This change is all that is required to correct that.